### DISCUSSION TITLE
Projects beta: More customization options for group by sort order.

### DISCUSSION THREAD

**Discussion by zorn** [community/community#7675] [Projects and Issues]
My main project has milestones:

* Private Alpha
* Public Beta 
* Launch 1.0

When I use the Group By Milestone, it lists them in an alphabetical order.

![Screenshot 2021-11-13 at 11-19-19 Build software better, together](https://user-images.githubusercontent.com/52168/141651119-f2414842-db1a-434f-8287-fd40d89a1f98.png)

I'd really prefer it to use the milestone dates, so I see it as: 

* Private Alpha
* Public Beta 
* Launch 1.0

**Comment by chikaramike** [comment]
Also a issue here: https://github.com/orgs/community/discussions/9595

**Comment by mouhamed1slem-bouazizi** [comment]
GitHub's milestone grouping uses alphabetical ordering by default, but you can work around this by using a naming convention that naturally sorts in chronological order like rename your milestones to include numbers:

1. Private Alpha
2. Public Beta
3. Launch 1.0

Or use dates in the milestone names:

2024-Q1: Private Alpha
2024-Q2: Public Beta
2024-Q3: Launch 1.0

Or use semantic versioning approach:

v0.1 - Private Alpha
v0.2 - Public Beta
v1.0 - Launch
